db/migrate/20161023011751_add_project_index_to_tasks_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddProjectIndexToTasksTable < ActiveRecord::Migration
+  def change
+    add_index :tasks, :project_id
```

Comment:
Not saying that there is a problem here at all, merely curious to hear your thoughts on why you've chosen to have a migration for the model, then separate migrations to both add the foreign key and add an index on the foreign key.